mesh/pmesh.hpp

```diff
@@ -244,6 +244,15 @@ class ParMesh : public Mesh
    /// Map a local element number to a global element number.
    long GetGlobalElementNum(int local_element_num) const;
 
+   /** The following functions define global indices for all local vertices,
+       edges, faces, or elements. The global indices have no meaning or
+       significance for ParMesh, but can be used for purposes beyond this class.
+    */
+   void GetGlobalVertexIndices(Array<HYPRE_Int> &gi);
+   void GetGlobalEdgeIndices(Array<HYPRE_Int> &gi);
+   void GetGlobalFaceIndices(Array<HYPRE_Int> &gi);
+   void GetGlobalElementIndices(Array<HYPRE_Int> &gi);
```

Comment:
Why aren't these functions `const`?

Reply:
These functions construct a `ParFiniteElementSpace` with `this` `ParMesh` as an argument, which is not `const`. Since `ParFiniteElementSpace` has been eliminated from `GetGlobalElementIndices`, that function only can be made `const`.